package.json

```diff
@@ -1,19 +1,13 @@
 {
   "name": "arkitektonika",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
The major version bump inclines incompatible API changes. Wouldn't the `ApiClient` of the [Arkitektonika Client](https://github.com/IntellectualSites/Arkitektonika-Client) at least need an update by then, considering it wraps around a versioned package.

Reply:
Good point, gonna use 1.1.0 then